### ISSUE TITLE
Small CSS color issue

### ISSUE THREAD

**Issue by vmario89** [manyfold3d/manyfold#3633]
Hi,

just a quick hint, which catches directly in the face:

![Image](https://github.com/user-attachments/assets/6099b835-097d-4c18-b9e1-95b784daf919)

the feature to select themes is very cool! I love it. I changed to some other and have seen that there is that blue box, which is not readable very well in the backend, as you can see. I guess some class is missing in stylesheet. Should maybe fixed with ease

best regards,
Mario

**Comment by Floppy** [comment]
Thanks! There will probably be a few of these with the new themes, so thank you very much for pointing it out :)
